src/CMakeLists.txt

```diff
@@ -52,6 +55,6 @@ install(TARGETS VectorSimilaritySpaces
     DESTINATION ${CMAKE_INSTALL_PREFIX})
 
 if(NOT VECSIM_STATIC)
-    set_target_properties(VectorSimilarity PROPERTIES PREFIX "")
+    set_target_properties(VectorSimilarity PROPERTIES PREFIX "lib")
```

Comment:
why the add?

Reply:
When I used scratch file for the benchmark, I couldn't link our VectorSimilarity shared library from command line. Then I understood from here https://www.cprogramming.com/tutorial/shared-libraries-linux-gcc.html that GCC assumes that all libraries start with lib and end with .so